modules/addressing-cuda-oom-errors.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+
+[id="addressing-cuda-oom-errors-for-the-single-model-serving-platform_{context}"]
+= Addressing CUDA Out of Memory errors
```

Comment:
A couple of things:

- I suggest using "resolving" instead of "addressing". That's the verb we use in our release notes.
 - At Red Hat, we use sentence-style capitalization for headings, not headline-style. See https://redhat-documentation.github.io/supplementary-style-guide/#titles-and-headings. 
 - In this case, "out of memory" is a compound adjective that describes "errors". So it should all be hyphenated.
 - Putting that all together: "Resolving CUDA out-of memory errors"